packages/BetterPhpDocParser/Printer/PhpDocInfoPrinter.php

```diff
@@ -93,6 +94,10 @@ public function printNew(PhpDocInfo $phpDocInfo): string
             return $this->docBlockInliner->inline($docContent);
         }
 
+        if ($phpDocInfo->getNode() instanceof InlineHTML) {
```

Comment:
this fix, fixed my actual real world use-case.

since I needed 2 fixes (see comment below), I guess either the unit test needs some adjustments or the actual fixing IF-statement needs to be relocated..